api/generic/insights.go

```diff
@@ -9,7 +9,10 @@ import (
 type Insight interface {
 	proto.Message
 	IsOnline() bool
+	// TODO Deprecated: bad
```

Comment:
```suggestion
	// Deprecated: bad
```

That way in IDEs there usages are strikethrough

Reply:
I want to remove it in this PR